ultradns-rest/src/test/java/denominator/ultradns/MockUltraDNSRestServer.java

```diff
@@ -0,0 +1,96 @@
+package denominator.ultradns;
```

Comment:
Please add some more tests for all the classes. Seems like the current UT code coverage is less than 40%.